.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,11 @@
+## What does this implement/fix? Explain your changes.
+<!-- Give as much information as possible. Explain your rationale to help reviewers understand what's being done. -->
+
+## Does this close any currently open issues?
+<!-- If there is none present, remove this section -->
+
+## Any relevant logs, error output, etc?
+<!-- If it’s long, please paste to https://gist.github.com/ and insert the link here. -->
+
+## Any other comments?
```

Comment:
One thing that could be good to add here is how to test this change? Is there a sequence of commands or actions that could be executed to verify if the PR works as intended?